index.html

```diff
@@ -215,6 +215,32 @@ <h3 id="aboutSelfTitle" class="aboutTitles defaultFontFamily">About Myself</h3>
         </li>
       </ul>
     </section>
+
+    <!-- CONTACT FORM -->
+    <section class="contactFormSection">
+      <div class="formHeadingContainer defaultFontFamily">
+      <h3>Contact me</h3>
+      <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+I’d love to help with it!</p>
+      </div>
+      <form action="https://formspree.io/f/xqkonnqr" method="POST" class="contactForm">
+        <ul>
```

Comment:
- [ ] Please avoid using the `ul` inside the form. You can still use the form and with the help of `flexbox` you can easily attain the structure. As I noticed that you used the `flexbox` on your `ul` but you can optimize it more and make your code semantically more correct which makes your website more accessible. Whereas please note that `ul` and `li` are just meant for listing purposes so use it wisely 😄 . If you do the changes your code looks more leaner and easy to understand.